docs/_snippets/builds/saving-data/manualsave.html

```diff
@@ -15,7 +15,7 @@
 
 <p>Server data:</p>
 
-<pre><code id="snippet-manualsave-console">&#x3C;p&#x3E;Change the content of this editor, then save it on the server.&#x3C;/p&#x3E;</code></pre>
+<pre class="highlight language-json" id="snippet-manualsave-console">&#x3C;p&#x3E;Change the content of this editor, then save it on the server.&#x3C;/p&#x3E;</pre>
```

Comment:
Is there a reason for using `language-json` not `language-html`?

Reply:
There's no reason. As far as I've tested this, they're both styled the same way.